csharp/ql/src/semmle/code/csharp/frameworks/system/Xml.qll

```diff
@@ -2,6 +2,7 @@
 
 import csharp
 private import semmle.code.csharp.frameworks.System
+private import semmle.code.csharp.dataflow.DataFlow2
```

Comment:
This has the potential to conflict with `Format.qll` as well as user-defined configurations. Use `DataFlow4` here instead?

Reply:
That actually wouldn't be problematic, as the two instances in XML.qll and Format.qll are unrelated (one is not used to define the other, and vice versa).